src/Symfony/Bridge/Doctrine/Tests/DataCollector/StringRepresentableClass.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Symfony\Bridge\Doctrine\Tests\DataCollector;
+
+/**
+ * A class for testing __toString method behaviour. It's __toString returns a value, that was passed into constructor.
+ */
+class StringRepresentableClass
+{
+    /**
+     * @var string
+     */
+    private $representation;
+
+    /**
+     * CustomStringableClass constructor.
+     *
+     * @param string $representation
+     */
+    public function __construct($representation)
+    {
+        $this->representation = $representation;
+    }
+
+    public function __toString()
+    {
+        return $this->representation;
```

Comment:
Can we simplify this class by removing the `representation` property and hardcode the text here directly?